Deployment/Milyli.ScriptRunner.Services.Interfaces.nuspec

```diff
@@ -2,6 +2,7 @@
 <package>
   <metadata>
     <id>Milyli.ScriptRunner.Services.Interfaces</id>
+	<version>1.0.0.0</version>
```

Comment:
Found out that this version tag is both required & that it;s contents will be replaced by whatever is passed to the `-Version` option when invoking the nuget `pack` command